src/main.cpp

```diff
@@ -1644,6 +1646,10 @@ void setup() {
     u8g2_prepare();
     displayLogo(String("Version "), String(sysVersion));
     delay(2000); // caused crash with wifi manager on esp8266, should be ok on esp32
+
+#if FEATURE_MENU == 1
```

Comment:
maybe you could be more specific  with the naming:
-> FEATURE_DISPLAY_MENU
-> displayMenuHandler